scripts/install_bb.py

```diff
@@ -32,224 +31,182 @@ def q0():
+        root_dict['v_mod_1d_name'] = v_mod_1d_path
+        # root_dict['hf_v_model_path']=v_mod_1d_path
+        root_dict['hf_stat_vs_ref'] = hf_stat_vs_ref
+        root_dict['bb']['rand_reset'] = True
+    else:
+        is_site_specific_id = q0()
         if is_site_specific_id:
-            v_mod_1d_path,hf_stat_vs_ref=q1_site_specific(os.path.dirname(stat_file))
+            v_mod_1d_path, hf_stat_vs_ref = q1_site_specific(os.path.dirname(params.stat_file))
             v_mod_1d_name = "Site_Specific"
-            params_base_bb_dict['site_specific']=True
-            params_base_bb_dict['hf_v_model_path']=v_mod_1d_path
-            params_base_bb_dict['hf_stat_vs_ref']=hf_stat_vs_ref
-            params_base_bb_dict['rand_reset']=True  
+            root_dict['bb']['site_specific'] = True
+            root_dict['v_mod_1d_name'] = v_mod_1d_path
+            # root_dict['hf_v_model_path']=v_mod_1d_path
+            root_dict['hf_stat_vs_ref'] = hf_stat_vs_ref
+            root_dict['bb']['rand_reset'] = True
 
         else:
-            v_mod_1d_name, v_mod_1d_selected = q1_generic(v_mod_1d_dir)
-            params_base_bb_dict['site_specific']=False
-            params_base_bb_dict['hf_v_model']=v_mod_1d_selected
-
-    params_base_bb_dict['v_mod_1d_name']=v_mod_1d_name
+            v_mod_1d_name, v_mod_1d_selected = q1_generic(V_MOD_1D_DIR)
+            root_dict['bb']['site_specific'] = False
+            root_dict['v_mod_1d_name'] = v_mod_1d_selected
+            # root_dict['hf_v_model']=v_mod_1d_selected
 
-
-    #globals_dict = globals()
+    # root_dict['v_mod_1d_name']=v_mod_1d_name
 
     if old_params:
-        hf_kappa_list = [hf_kappa]*len(srf_files)
-        hf_sdrop_list = [hf_sdrop]*len(srf_files)
-        if len(srf_files) > 1:
+        # hf_kappa_list = [params.hf.hf_kappa]
+        # hf_sdrop_list = [params.hf.hf_sdrop]
+        if len([params.srf_file]) > 1:
             print "Info: You have specified multiple SRF files."
-            print "      A single hf_kappa(=%s) and hf_sdrop(=%s) specified in params.py will be used for all SRF files." %(hf_kappa, hf_sdrop)
+            print "      A single hf_kappa(=%s) and hf_sdrop(=%s) specified in params.py will be used for all SRF files." % (
+                params.hf.hf_kappa, params.hf.hf_sdrop)
             print"       If you need to specific hf_kappa and hf_sdrop value for each SRF, add hf_kappa_list and hf_sdrop_list to params_base.py"
 
-    else:
-        print "hf_kappa_list: ", hf_kappa_list
-        print "hf_sdrop_list: ", hf_sdrop_list
-        print "srf_files:", srf_files
-        if len(hf_kappa_list) != len(hf_sdrop_list) or len(hf_kappa_list) != len(srf_files):
-            print "Error: hf_kappa_list (len=%d), hf_sdrop_list (len=%d) and srf_files (len=%d) should be of the same length."%(len(hf_kappa_list),len(hf_sdrop_list), len(srf_files))
-            sys.exit()
+    #dunno what the following does, old_prams always=True
```

Comment:
You should be more responsible and ask around for answers before you made drastic changes.
The 'old_params' was there because at one point we decided to support both 'single' and 'multiple' SRF per simulation. 
But now since we are enforcing single SRF only, any related script shuold be removed.